### ISSUE TITLE
notices on validate() when no context chains are given.

### ISSUE THREAD

**Issue by NickBelhomme** [particle-php/Validator#53]
# Context

When initiating a clean validator without attaching chains you get notices.

Why would you do this? To provision a clean validator and run a valid on it.
This is done when you provide a template method in which the children can define the chains (+ rules).

When the child class does not implement the template method it should not throw any notices.

The outcome TRUE is actually good in my opinion.

# reproduction

```
$validator = new \Particle\Validator\Validator();
var_dump($validator->validate(['money' => '850.00']));
```

# current result

PHP Notice:  Undefined index: default in ParticleValidator/src/Validator.php on line 110
PHP Warning:  Invalid argument supplied for foreach() in ParticleValidator/src/Validator.php on line 110
bool(true)

**Comment by berry-langerak** [comment]
Ouch. Will fix asap. :)

**Comment by berry-langerak** [comment]
Has been fixed in https://github.com/particle-php/Validator/pull/54

**Comment by RickvdStaaij** [comment]
Fix is merged, closing issue.

**Comment by RickvdStaaij** [comment]
Also, fix is released in version `v1.0.2`
